kotlin-mbedtls-metrics/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
Idea for separate ticket: we could move this to common parts in root build.gradle.kts under subproject section.